ui/src/containers/AppTableContainer.js

```diff
@@ -19,25 +19,25 @@ export class AppDetailsContainer extends React.Component {
 
   getTable = () => {
     const renderedRows = [];
-    for (let i = 0; i < this.props.apps.length; i++) {
+    for (let i = 0; i < this.props.app.length; i++) {
       const tempRow = [];
-      tempRow[0] = this.props.apps[i][0];
-      tempRow[1] = this.props.apps[i][1];
-      tempRow[2] = this.props.apps[i][2];
-      tempRow[3] = this.props.apps[i][3];
+      tempRow[0] = this.props.app[i][0];
+      tempRow[1] = this.props.app[i][1];
+      tempRow[2] = this.props.app[i][2];
+      tempRow[3] = this.props.app[i][3];
       const cb = (
         <React.Fragment>
           <Checkbox
             label=""
-            isChecked={this.props.apps[i][4]}
+            isChecked={this.props.app[i][4]}
             onChange={this.handleChange}
             aria-label="controlled checkbox example"
```

Comment:
We probably want to change this label as well.

Reply:
good spot :+1: